symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/DatafeedLoopV1.java

```diff
@@ -114,7 +114,11 @@ private Void readAndHandleEvents() throws ApiException {
         datafeedApi.v4DatafeedIdReadGet(datafeedId, authSession.getSessionToken(), authSession.getKeyManagerToken(),
             null);
     if (events != null && !events.isEmpty()) {
-      handleV4EventList(events);
+      try {
+        handleV4EventList(events);
+      } catch (RequeueEventException e) {
+        log.warn("EventException is not supported for DFv1, events will not get re-queued", e);
```

Comment:
Is it something we want to log at `warn` level ?

Reply:
I would say yes because it won't prevent the bot to work (or from the bdk point of view there is nothing that can be done to recover from that) but might lead to unexpected behavior

Reply:
or were you thinking about lowering the level to info?